04_numeronym/vinilapeddireddy/main_test.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var tests = []struct {
+	inputStrings  []string
+	outputStrings []string
+}{
+	{[]string{}, nil},
+	{[]string{"accessibility", "Kubernetes", "abc", "charecters"}, []string{"a11y", "K8s", "abc", "c8s"}},
+	{[]string{"!!&^#*&+"}, []string{"!6+"}},
```

Comment:
Nice, could we have a test with spacd and unicode as well please?

Reply:
Done